### ISSUE TITLE
Deixar mais claro o que é o projeto

### ISSUE THREAD

**Issue by Ascenio** [InterruptorPt/ate-onde-chega-cultura#55]
A intenção do projeto parece estar bem explícita, porém do ponto de vista tecnológico nem tanto. Na raiz do projeto você tem vários arquivos HTML, mas também tem uma pasta de um app mobile, não fica claro se é um projeto Web ou Mobile (ou ambos). Seria bom deixar mais claro isso no `README.md` principal, dizer que existem dois subprojetos etc. Além disso, deveria organizar os arquivos em pasta por subprojeto, por exemplo criar uma pasta para Web.

**Comment by marado** [comment]
Também seria útil clarificar no README que este projecto só "visualiza dados" que vêm do wikidata.

**Comment by opedromandrade** [comment]
> Também seria útil clarificar no README que este projecto só "visualiza dados" que vêm do wikidata.

Concordo. estava a ver de onde vinham os para tentar actualizar na zona onde resido e me desloco, e apenas consegui perceber quando li o que escreveste. temos portanto para colaborar na actualização de dados, colaborar no wiki data, correcto?
cheers.

**Comment by marado** [comment]
Os dados têm primeiro de ser actualizados no wikidata, sim.

**Comment by opedromandrade** [comment]
> Os dados têm primeiro de ser actualizados no wikidata, sim.

obrigado pelo heads up. em loures, já começarm a surgir mais equipamentos culturais no wikidata e open street maps.
